app/frontend/src/registry/components/people/PersonDetail.vue

```diff
@@ -580,6 +580,8 @@ export default {
         this.closeApplication(key)
         this.savingApplication = false
         this.updateRecord()
+      }).catch(() => {
+        this.savingApplication = false
```

Comment:
We need to discuss a logging framework. Ignoring exceptions is bad. We should be doing something along the lines of:
`logger.error(e)`
Ensuring that errors like these end up with our log aggregator, end get logged to consoles when were in development mode.

Could we for now at least do a console.error(e) so that we don't just ignore it?